GNUmakefile

```diff
@@ -145,17 +145,19 @@ gen:
 	rm -f internal/conns/*_gen.go
 	rm -f internal/provider/*_gen.go
 	rm -f internal/service/**/*_gen.go
-	rm -f internal/sweep/sweep_test.go
+	rm -f internal/sweep/sweep_test.go internal/sweep/service_packages_gen_test.go
```

Comment:
Are the `rm` commands needed? In most cases, the generators should be truncating the files

Reply:
Most likely not, but keeping them for now for consistency.